clang-tools-extra/docs/clang-tidy/checks/misc/must-use.rst

```diff
@@ -0,0 +1,28 @@
+.. title:: clang-tidy - misc-must-use
+
+misc-must-use
+=============
+
+Allows for strictly enforce variables are used for specific classes, even with
+they would not be normally warned using -Wunused-variable due to templates or
+custom destructors.
+
+In the following example, future2 normally would not trigger any unused variable checks,
```

Comment:
Please highlight `future2` with double back-ticks. Same for other occurrence.

Reply:
Thanks done.